components/clock.rst

```diff
@@ -75,6 +75,29 @@ When using the Clock component, you manipulate
 :class:`Symfony\\Component\\Clock\\DatePoint` instances. You can learn more
 about it in :ref:`the dedicated section <clock_date-point>`.
 
+Storing DatePoints in Databases
+-------------------------------
+
+If you :doc:`use Doctrine </doctrine>`, consider using the ``date_point`` Doctrine
+type, which converts to/from DatePoint objects automatically::
+
+    // src/Entity/Product.php
+    namespace App\Entity;
+
+    use Doctrine\ORM\Mapping as ORM;
+    use Symfony\Bridge\Doctrine\Types\DatePointType;
+    use Symfony\Component\Clock\DatePoint;
+
+    #[ORM\Entity(repositoryClass: ProductRepository::class)]
+    class Product
+    {
+        #[ORM\Column(type: DatePointType::NAME)]
```

Comment:
```suggestion
        #[ORM\Column(type: DatePointType::class)]
```
couldn't we use `::class` here? 🤔

Reply:
I'd even remove `type` since it's auto-detected from the property type